task/06-conditions-n-loops-tasks.js

```diff
@@ -419,13 +386,7 @@ function isBracketsBalanced(str) {
             }
         }
     }
-
-    if (arrTemp.length != 0) 
-    {
-        return false;
-    }
-
-    return true;
+    return (arrTemp.length == 0);
```

Comment:
operator === is always preferable
sometimes ''!=' may cause an error
(check at console :
"0" == 0
and "0" === 0)

Reply:
Fixed comment.